core/.stylelintrc.yml

```diff
@@ -7,6 +7,7 @@ ignoreFiles:
   - src/css/flex-utils.scss
   - src/css/normalize.scss
   - src/css/text-alignment.scss
+  - src/css/visibility.scss
```

Comment:
Can you change this line to `display` please 🙂

Reply:
You got it! Didn't even know it was there. Thanks!